packages/messages/message-box.scss

```diff
@@ -0,0 +1,116 @@
+@import "~@fremtind/jkl-core/variables/_all.scss";
+
+$colored-line-width: $component-spacing--xs;
+$message-width: rem(560px);
+
+.jkl-message-box {
+    position: relative;
+    width: 100%;
+    max-width: $message-width;
+    min-height: $layout-spacing--xl;
+    padding: $component-spacing--xl;
+    padding-left: rem(56px);
+    background-color: $gra-10;
+    box-sizing: border-box;
+
+    &__icon {
+        position: absolute;
+        left: $spacing-5;
+    }
+
+    &__title {
+        margin-bottom: $component-spacing--large;
+    }
+
+    &__message,
+    &__message:last-child {
+        margin-bottom: 0;
+    }
+
+    &--full {
+        max-width: 100%;
+    }
+
+    &--info {
+        border: rem(1px) solid $info;
+        background-color: change-color($info, $lightness: 97%);
```

Comment:
Ble ikke denne lagret som en variabel et eller annet sted?

Reply:
Nei, tror aldri det skjedde. Sikkert en god idé å gjøre!

Reply:
Godt innspill. Lager en `light` og `dark` mixin, som tar fargen som input.

Reply:
Disse er vel spesifikke farger som er regnet ut fra "hoved"-fargene. Jeg tror derfor de hører hjemme som definerte fargevariabler, enten her i komponenten hvis de kun brukes her, eller i `core` om de også brukes andre steder.

Reply:
F.eks.
```scss
$info-bakgrunn: change-color($info, $lightness: 97%);
$info-bakgrunn--darkbg: change-color($info--darkbg, $lightness: 5%);
```

Reply:
Sjekket om de var brukt andre steder, og så ut som det kun var gjort i messages og message-box. Så da gikk jeg litt lenger enn kanskje strengt tatt nødvendig: https://github.com/fremtind/jokul/pull/923/commits/ed2f5d724e05f9351612a7ca3658a5e3ce5a461e
Er det for voldsom abstrahering igjen? 🤔

Reply:
Er kanskje ikke merged enda, men samme verdier var brukt i PRen til Eduardo